src/cmd/build.js

```diff
@@ -131,7 +131,8 @@ async function defaultPackageCreator({
   await eventToPromise(stream, 'close');
 
   if (showReadyMessage) {
-    log.info(`Your web extension is ready: ${extensionPath}`);
```

Comment:
A generated zip should work fine in Firefox without signing it (as long as signing is disabled and bugs are fixed!) so I'm hesitant to add a note like this. There are a lot of use cases for the `build` command which I started listing at the top of this thread. 

Maybe it would just be more helpful to fix the bug in Firefox that initially caused confusion? It might help to comment on that bug to try and convince them why it's important.